platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -39,7 +39,13 @@ private Mapbox(@NonNull Context context, @NonNull String accessToken) {
    * @return Mapbox Access Token
    */
   public static String getAccessToken() {
-    return INSTANCE.accessToken;
+    String accessToken = INSTANCE.accessToken;
+    // validate if accessToken is correct
```

Comment:
Just a small suggestion for clarity, could we extract the validation code into a `private void validateAccessToken()` method inside `Mapbox` so that the `getAccessToken()` is simply `validateAccessToken(INSTANCE.accessToken); return INSTANCE.accessToken;`?